vignettes/VISION-vignette.Rmd

```diff
@@ -172,3 +172,78 @@ projection <- read.csv("umap_results.csv")
 
 vis <- addProjection(vis, "UMAP", projection)
 ```
+
+## Reproducing a plot from an interactive plot
+
+As of version 3.0.0, we have enabled users to save the relevant information from an interactive report to reproduce in R. This can be accomplished by downloading the "state" of the report using the `Download` button below "Save Report Info" in the upper right of the interactive web-based report. Below, we provide an example workflow of doing this.
+
+Clicking `Download` will download three files: (1) a DE cache; (2) a cache for selections; and (3) a json file containing the current projection, values being plotted, and selected cells. If you would like to view the state on the UI, you may read in the first two objects into VISION and launch the report again:
+
+```{r, collapse=T, message=F, results=F, eval=F}
+vis_new <- load_de_cache(vis, "de_cache_download_path")
+vis_new <- load_selections(vis, "selections_download_path")
```

Comment:
vis in the argument in second line has to be vis_new

Reply:
Good call.